064_arrayReverse.js

```diff
@@ -0,0 +1,9 @@
+const arrayReverse = (arr) => {
+  let reverse = [];
```

Comment:
This is not inplace reversal. You have to replace without using extra space

Reply:
Will do